lib/avro_turf/cached_schema_registry.rb

```diff
@@ -1,7 +1,10 @@
+require 'avro_turf/schema_registry'
+
 # Caches registrations and lookups to the schema registry in memory.
-class AvroTurf::CachedSchemaRegistry
+class AvroTurf::CachedSchemaRegistry < DelegateClass(AvroTurf::SchemaRegistry)
```

Comment:
I'd actually prefer to duplicate the methods, or just call them directly on the upstream.